docs/usage/customization-guide.md

```diff
@@ -491,6 +491,42 @@ rules to use. In other words, these are variables that are not advertised as
 node labels. See [backreferences](#backreferences) for more details on the
 usage of vars.
 
+#### Extended resources
+
+The `.extendedResources` field is a list of extended resources to advertise.
+See [extended resources](#extended-resources) for more details.
+
+Take this rule as a referential example:
+
+```yaml
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: my-sample-rule-object
+spec:
+  rules:
+    - name: "my sample rule"
+      extendedResources:
+        - "cpu-security.tdx.total_keys"
+      matchFeatures:
+        - feature: cpu-security.tdx.enalbed
```

Comment:
From the documentation here https://kubernetes-sigs.github.io/node-feature-discovery/v0.12/usage/features#cpu it looks like `cpu-security.tdx.enabled` is a boolean, so how it will be converted to`31`?

Reply:
@PiotrProkop, the example given may not have been the best, as I'm working on exposing the TDX values as part of https://github.com/kubernetes-sigs/node-feature-discovery/pull/1079

Reply:
jejeje yeah, I used a non-merged PR for the example, my bad 😅